### PR TITLE
fix: email_trim_fix

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res [hunk 1]
```diff
@@ -36,7 +36,19 @@ let emailField = FormRenderer.makeFieldInfo(
   ~name="email",
   ~placeholder="Enter your Email",
   ~isRequired=false,
-  ~customInput=InputFields.textInput(~autoComplete="off", ()),
+  ~customInput=(~input, ~placeholder as _) =>
+    InputFields.textInput(
+      ~input={
+        ...input,
+        onChange: event => {
```
Comment (PritishBudhiraja) [inline src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res:43]:
```Suggestion
        onChange: event =>
          (event->ReactEvent.Form.target)["value"]
          ->String.trim
          ->Identity.stringToFormReactEvent
          ->input.onChange,
      },

Comment (PritishBudhiraja) [reply src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res:43]: Curly brackets also not needed

File: src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res [hunk 2]
```diff
@@ -36,7 +36,20 @@ let emailField = FormRenderer.makeFieldInfo(
...
+        onChange: event =>
+          ReactEvent.Form.target(event)["value"]
+          ->String.trim
+          ->Identity.stringToFormReactEvent
+          ->input.onChange,
+      },
+      ~placeholder="Enter your Email",
+      ~autoComplete="off",
+      (),
+    ),
```
Comment (JeevaRamu0104) [inline src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res:52]: try adding test cases for this

**Review by PritishBudhiraja — DISMISSED** [review]

File: src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res (hunk 2 above)
Comment (gitanjli525) [reply src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res:52]: @JeevaRamu0104 added the test case for same

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**